hack/deploy-logging.sh

```diff
@@ -140,20 +145,20 @@ if [ ! -d $ARTIFACT_DIR ] ; then
     mkdir -p $ARTIFACT_DIR
 fi
 
-if [ "${USE_OLM:-false}" = true ] ; then
-    deploy_logging_using_olm
-else
-    ESO_NS=openshift-logging
-    deploy_logging_using_clo_make
-fi
-
 # Create the $LOGGING_NS namespace:
 if oc get project $LOGGING_NS > /dev/null 2>&1 ; then
     echo using existing project $LOGGING_NS
 else
     oc create -f $TEST_OBJ_DIR/openshift-logging-namespace.yaml
 fi
 
+if [ "${USE_OLM:-false}" = true ] ; then
+    deploy_logging_using_olm
+else
+    ESO_NS=openshift-logging
```

Comment:
so that this matches what we'd see in a typical cluster deployment can we make this `openshift-operators-redhat` instead?

Reply:
then it won't match the cluster-logging-operator `make deploy`